golangci.yaml

```diff
@@ -1,12 +1,35 @@
-linters:
-  disable:
-    - megacheck
+liners:
+  disable-all: true
   enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
+    - goprintffuncname
+    - gosec
     - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - nakedret
+    - nolintlint
+    - rowserrcheck
+    - scopelint
     - staticcheck
+    - structcheck
+    - typecheck
+    - unconvert
+    - varcheck
+    - gocritic
     - unused
-    - interfacer
-    - errcheck
-    - prealloc
-    - gosec
+    - misspell
+    - dupl
 
+# Consider enabling these, some cleanup required:
+#    - stylecheck  # Replacement for golint, valid but numerous.
+#    - gomnd       # Avoid magic numbers.
+#    - whitespace  # Consistent use of whitespace.
```

Comment:
I suggest to add `wsl` to the list above instead of `whitespace` only.

Reply:
+1